src/content/release/breaking-changes/default-systemuimode-edge-to-edge.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Default `SystemUiMode` set to Edge-to-Edge
+description: >
+    Apps targeting Android SDK 15+ will be opted
+    into edge-to-edge by default.
+---
+
+## Summary
+If your Flutter app runs on Android and targets SDK version 15 or above,
+your app will automatically be displayed in [edge-to-edge mode][1] on
+devices running Android SDK 15+. In order to maintain your current
+non-edge-to-edge app layout (including an unset `SystemUiMode`),
+follow the [migration guide](#migration-guide) below.
+
+## Context
+Android is enforcing [edge-to-edge mode][2] by default for all apps targeting
+Android 15 and above; see the [Android release notes][3] for more details on
+this change. As a result, all Flutter apps running on Android that target
+Android 15 and above will be opted into edge-to-edge mode by default, impacting
+devices running on Android SDK 15+/API 35+ just like any other Android app.
+
+At the time of publishing this guide, Flutter apps target Android 14 by
+default and will not be opted into edge-to-edge mode automatically, but
+your app will be impacted when you choose to target Android 15. If your app
+targets `flutter.targetSdkVersion` (as it does by default), then your app will
+target Android 15 starting with Flutter version 3.26, automatically opting your
+app into edge-to-edge as a result; see the [timeline](#timeline) below for
+details. If your app explicitly sets `SystemUiMode.edgeToEdge` for it to run in
+edge-to-edge mode via a call to [`SystemChrome.setEnabledSystemUIMode`][4],
+then your app is already migrated. Apps needing more time to migrate to
+edge-to-edge mode will need to follow the steps below to opt out on
+devices running Android SDK 15+.
+
+Please note that 
+
+1. Android plans for the workaround detailed below to be temporary.
+2. Flutter plans to align with Android (and iOS) to
+support edge-to-edge by default within the year, so **please migrate to**
+**edge-to-edge before the operating system removes the ability to opt out**.
+
+## Migration Guide
+To opt out of edge-to-edge by default on SDK 15, a new style attribute
+specification is required in each activity that needs an opt out. If
+you have a parent style that all other styles that need an opt out inherit
+from, then you can modify the parent only. In the example below, you will
+update the style generated from `flutter create`.
```

Comment:
Our docs use sentence casing for titles and anchors.
We avoid using "above" or "below" in text. If it's translated to Chinese or Hebrew, for example,
above/below might incorrect. Another general writing rule.

```suggestion
## Migration guide

To opt out of edge-to-edge on SDK 15, specify the new style attribute
in each activity that requires it. If you have a parent style that child styles
need to opt out of, you can modify the parent only.
In the following example, update the style generated from `flutter create`.
```